frontend/src/component/app/app.js

```diff
@@ -4,6 +4,7 @@ import { Provider } from 'react-redux';
 
 import LandingContainer from '../landing/landing';
 import DashboardContainer from '../dashboard/dashboard';
+import SettingsContainer from '../settings/settings';

```

Comment:
There is no `settings/` directory. Is this legacy code? I deleted this line and the line below (line 43) that 'uses' it, and then the code worked. It was broken if I left it in.